.gometalinter.json

```diff
@@ -1,15 +1,22 @@
 {
     "Exclude": [
         ".*_mock.go",
-        "geth/jail/doc.go"
+        "geth/jail/doc.go",
+        ".*Errors unhandled.*fmt.Fprintf.*"
```

Comment:
Can we also make sure that it is `gas`? It should print out its name too.

Reply:
Sure.